helpers/deproxy.py

```diff
@@ -406,6 +415,11 @@ def parse_firstline(self, stream):
         if requestline[-1] != '\n':
             raise IncompleteMessage('Incomplete request line!')
 
+        # Skip optionsl empty lines
+        while not re.match('[A-Za-z]', requestline) and len(requestline) > 0:
```

Comment:
```suggestion
        while re.match('^$', requestline):
```
to skip exactly empty lines? In your case you'll also skip lines like "234.."